action.yml

```
@@ -220,7 +220,7 @@ runs:
       if: ${{ steps.infrastructure-changes.outputs.terraform == 'true' && inputs.staging == 'true' }}
       working-directory: terraform
       run: |
-        terraform init -backend-config="key=staging/ecs_service_theme/terraform.tfstate"
+        terraform init -backend-config="key=staging/ecs_service_${{ env.SERVICE_NAME }}/terraform.tfstate"
```

I think a search and replace is safer
```suggestion
        staging_env_key=$(sed -n 's/^.*production\/\(.*\)\"$/staging\/\1/p' backend.tf)
        terraform init -backend-config="key=${staging_env_key}"
```

---

I think we will force consistency in this way to have `ecs_service_` as prefix

---

Then it's awkward for theme-service because we have to use `ecs_service_theme_service`

---

hmmm, I'll create a ticket to fix theme service. I'd say all our services should ideally have -service at the end